cluster/BUILD

```diff
@@ -46,3 +46,33 @@ pkg_tar(
         "//cluster/saltbase:salt-manifests",
     ],
 )
+
+sh_test(
+    name = "common",
```

Comment:
may want to suffix all of these with `_test`. otherwise you have a weird target named `//cluster:common` which is actually a test, for example.